examples/tilted_bottom_boundary_layer.jl

```diff
@@ -98,10 +98,17 @@ coriolis = ConstantCartesianCoriolis(f = 1e-4, rotation_axis = ĝ)
 N² = 1e-5 # s⁻² # background vertical buoyancy gradient
 B_field = BackgroundField(constant_stratification, parameters=(; ĝ, N² = N²))
 
-# Because the bottom boundary condition is that there must be zero *total* diffusive
-# flux across the seafloor, the default `FluxBoundaryCondition()` is insufficient;
-# instead, the boundary condition on the perturbation flux must take the background
-# into account,
+# We choose to impose a bottom boundary condition of zero *total* diffusive buoyancy
+# flux across the seafloor,
+# ```math
+# ∂_z B = ∂_z b + N^{2} \cos{\theta} = 0.
+# ```
+# The default `FluxBoundaryCondition()` (which imposes ``∂_z b = 0``) is inappropriate;
+# instead, we must rearrange this expression to override the boundary condition on the
+# perturbation flux:
+# ```math
+# ∂_z b = - N^{2} \cos{\theta}.
+#```
```

Comment:
```suggestion
# This shows that to impose a no-flux boundary condition on the total buoyancy field ``B``, we must apply a boundary condition to the perturbation buoyancy ``b``,
# ```math
# ∂_z b = - N^{2} \cos{\theta}.
#```
```

Reply:
I'm just suggesting a small change in tone here --- setting boundary conditions should always be an intentional act, even if using the defaults (this is one the arguments against having a default at all...) I don't think we want to think of setting boundary conditions as an "override".